doc/source/whatsnew/v2.0.1.rst

```diff
@@ -20,10 +20,12 @@ Fixed regressions
 
 Bug fixes
 ~~~~~~~~~
+- Bug in :meth:`Series.concat`, Attribute Error raised with a None and Timestamp (:issue:`52093`)
```

Comment:
```suggestion
- Bug in :meth:`pd.concat` where an ``AttributeError`` raised with a None and Timestamp (:issue:`52093`)
```

Reply:
Also sorry could you move this to `v2.1.0.rst` instead?

Reply:
done